### PR TITLE
Allow .post releases

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

File: get_releasenote.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def parse(
     msg = msg.strip()
     head_re = re.compile(
         head_line.format(
-            version="(?P<version>[0-9][0-9.abcr]+)",
+            version="(?P<version>[0-9][0-9.abcr]+(.post[0-9]+)?)",
```
Comment (webknjaz) [inline get_releasenote.py:30]: Shouldn't `.` be escaped as `\.`?

Comment (webknjaz) [reply get_releasenote.py:30]: Also, a full regex is a bit more complicated: https://www.python.org/dev/peps/pep-0440/#appendix-b-parsing-version-strings-with-regular-expressions.

**Review by webknjaz — APPROVED** [review]

**Comment by Dreamsorcerer** [comment]
Need to release as well: #15

**Comment by asvetlov** [comment]
@Dreamsorcerer the PR is correct, but I would say that `.post` is not SemVer compatible.
See the bottom of https://semver.org/ page for regexes and links to regex101.com for experimenting.

It leads to a confusing situation when Dependabot parses the post-release version as a pre-release that is *earlier* than the final.
Worse, the same is true for GitHub security checkers.
I suggest publishing the next *patch release* instead of *post-release* to don't shut the leg.

**Comment by webknjaz** [comment]
FWIW post and non-post release pairs don't normally have functional differences so most of the time it doesn't really matter much if somebody uses it.

**Pull request merged**